utils/umaApi.ts

```diff
@@ -230,5 +230,5 @@ export async function fetchCompleteSynth<T extends { type: ContractType }>(
 
 // longTokenName can return undefined if value wasn't found in API so we need to do a null check.
 export function formatLSPName(longTokenName: string): string {
-  return longTokenName.substring(-2);
+  return longTokenName?.substring(-2);
```

Comment:
When running the application locally, this variable was undefined for a particular address, causing the app to crash